fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -295,6 +332,16 @@ if [ ! -e "$fMRIFolder" ] ; then
 fi
 cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
 
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction"
+    #  use FSL's fslreorient2std for reorienting the image to match the approximate orientation of the standard template images MNI152
+    #  this makes the single-band processing more robust to registration problems
+    fslreorient2std "$fMRIFolder"/"$OrigTCSName".nii.gz "$fMRIFolder"/"$OrigTCSName"_prestcor.nii.gz
```

Comment:
It took me a while to parse "prestcor" to pre-st-cor, perhaps add some underscores or capitals.  Alternatively, use something like `_orig_reorient` to describe what was done, rather than what is expected to be done next.

Reply:
Noted.

Reply:
... and done.